jellyfin-api/src/jvmMain/kotlin/org/jellyfin/sdk/api/client/KtorClient.kt

```diff
@@ -105,7 +65,7 @@ public actual open class KtorClient actual constructor(
 		queryParameters: Map<String, Any?>,
 		requestBody: Any?,
 	): Response<T> {
-		val url = createUrl(pathTemplate, pathParameters, queryParameters)
+		val url = UrlBuilder.buildUrl(requireNotNull(baseUrl), pathTemplate, pathParameters, queryParameters)
```

Comment:
Shouldn't this still use `createUrl` but from the ApiClient interface?

Reply:
Hmm actually it should, I removed createUrl at first but added it back since the api's url functions use it